sdk/src/azure/core/az_mqtt5_rpc.c

```diff
@@ -21,6 +23,249 @@ static const az_span command_name_key = AZ_SPAN_LITERAL_FROM_STR(AZ_MQTT5_RPC_CO
 static const az_span executor_client_id_key
     = AZ_SPAN_LITERAL_FROM_STR(AZ_MQTT5_RPC_EXECUTOR_ID_KEY);
 static const az_span invoker_client_id_key = AZ_SPAN_LITERAL_FROM_STR(AZ_MQTT5_RPC_CLIENT_ID_KEY);
+static const az_span token_format_end_character = AZ_SPAN_LITERAL_FROM_STR("}");
+static const az_span token_topic_end_character = AZ_SPAN_LITERAL_FROM_STR("/");
+
+static const uint32_t az_mqtt5_rpc_client_id_hash = 3426466449;
```

Comment:
Can these be calculated in case the keys change again?

Reply:
Yes, you'd need to use `_az_mqtt5_rpc_calculate_hash` to re-calculate them

Reply:
Please add clear comments on what exactly each of these mean as well as how to calculate them.

Reply:
Also, please add simple UTs to ensure the hashes correspond to the expected strings.